dags/load_omni_data.py

```diff
@@ -42,25 +42,64 @@ def load_bobs():
 
     df =pd.DataFrame(result)
     df.columns=['id','mean', 'median','bob_lower','bob_upper', 'created_at', 'updated_at', 'bob_id', 'parameter_id']
+    
+    #I had a hard time getting rid of the uuid format so saving to csv and loading again was the workaroud
 
     df.to_csv('data.csv')
 
 
     df2=pd.read_csv('data.csv')
 
-
+    #change parameter_id names to representative names for easier processing
     df2=df2.replace(to_replace=['29fa6a90-ca21-40ef-8a72-f7035aa69226','d52bbcba-f1f6-4885-8510-dbaa6f91a4e0','fcd34841-d2b5-46d5-bacc-0a934412b491','78f330e9-8900-45e4-aa76-1bc412d4d65d'],
     value=['CalibrationDriveSpeedPercent','FormingDriveSpeedPercent' ,'WeldCurrent','SpeedDifferencePercent'],
     )
 
     df3=df2.pivot(index='created_at', values=['median','bob_lower','bob_upper'], columns='parameter_id')
-
+    
+    #rename colums
 
     d= df3.columns.swaplevel().map('_'.join)
 
     df3.columns=df3.columns.droplevel(0)
 
     df3.columns = d
 
-
+    #this is the directory where the expectations suite will go validate
     df3.to_csv('/home/ncamiso.khanyile/Data/bob_bobstats/bobs.csv')
+ 
+def load_prescribe_data():
+
+    model_query = """
+    select id as id, bob_upper as bob_upper, bob_lower as bob_lower,bob_median as bob_median,bob_target as bob_target,parameter_id as parameter_id
+    from prescribe_prescribereportdata 
+    where parameter_id =UUID('78f330e9-8900-45e4-aa76-1bc412d4d65d') or
+    parameter_id =UUID('d52bbcba-f1f6-4885-8510-dbaa6f91a4e0')
+    or parameter_id =UUID('fcd34841-d2b5-46d5-bacc-0a934412b491')
+    or parameter_id =UUID('29fa6a90-ca21-40ef-8a72-f7035aa69226')
+    limit 8000;;
+
+    """
+
+
+    with omni.conn as cur:
+        result = cur.execute(model_query).fetchall()
+
+
+
+    df =pd.DataFrame(result,columns=['id','bob_upper', 'bob_lower', 'bob_median','bob_target', 'parameter_id'])
+
+    df.to_csv('prescribe.csv')
+
+    df2=pd.read_csv('prescribe.csv')
+
+    df2=df2.replace(to_replace=['29fa6a90-ca21-40ef-8a72-f7035aa69226','d52bbcba-f1f6-4885-8510-dbaa6f91a4e0',
```

Comment:
This should be done in sql using the parameter_parameter table to return parameter titles, and outer joining with the raw values you selected earlier to add the titles to the raw df.